src/tutorials/0005/07-exercise.md

```diff
@@ -0,0 +1,17 @@
+Using the `cat` method, return the text contents of a file named `success.txt`, stored inside a `dir` directory which lives directly within a root directory.
+
+The CIDs associated with these directories are:
+
+* root directory: `"QmX1rvLYrhqfnnjvrFqudYZgQyomZxS9U9p5e8Dn3ot4Jk"`
+* `dir` directory: `"QmPT14mWCteuybfrfvqas2L2oin1Y2NCbwzTh9cc33GM1r"`
+
+Since you don't know the CID of the file itself, you'll need to define the its IPFS path by combining a directory's CID and the file's path relative to that directory.
+
+Based on our file structure, the path could be expressed as:
+
+* `dir/success.txt`, relative to the root directory
```

Comment:
I support this idea but it will require some changes under the hood to make a new file with a new message hidden it that's used in lessons 7 and 8 and doesn't take up too much space when the buffer is shown on the screen in lesson 8. 

Are fun emojis super space consuming? Or  how about including this kind of emoji-ish thing in a more fun version?--->   ¯\_(ツ)_/¯ 

 I don't have time to get to it today so I'll leave this to @dominguesgm for early Monday.